contracts/entry/library.cairo

```diff
@@ -14,13 +14,22 @@ namespace Entries:
     #
     # Helpers
     #
+
+    # @notice Aggregates entries for a specific value
+    # @param num_entries: number of entries to use for aggregation
+    # @param entries_ptr: entries list to use for aggregation
```

Comment:
Think we should group these two parameters and should rename them to `entries_len, entries` given the standard of representing lists this way

Reply:
Done in https://github.com/42labs/Empiric/pull/152/commits/91134dc1d73222dc92a54ec58ed4539c7c9efade